### PR TITLE
Open scholarship application and portfolio management

### PR DESCRIPTION
**Pull request by johnbyrne727rit**
As an administrator, I want to open the scholarship applications so that submitters can access their portfolio management and apply for scholarships.

Adds page to open scholarship portfolio period, as well as the portfolio period table on the database. The query file and mutation file started here will likely have to be modified to complete other tasks, this just does the starting setup (no association with other tables yet).

[Trello Card](https://trello.com/c/PLJXxhDr/43-open-scholarship-application-and-portfolio-management)

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   ],
   "scripts": {
     "start": "npm run build && node ./build/main.js",
+    "debug": "npm run build && node --inspect ./build/main.js",
```
Comment (johnbyrne727rit) [inline backend/package.json:31]: This is a new script I've been using that lets the chrome debugger attach to the BE. Here's a link to an [article for reference](https://medium.com/@paul_irish/debugging-node-js-nightlies-with-chrome-devtools-7c4a1b95ae27).

File: frontend/src/Admin/queries/portfolioPeriods.graphql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    description
+    numPieces
+    entryStart
+    entryEnd
+    judgingStart
+    judgingEnd
+    createdAt
+    updatedAt
+    judges {
+      username
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/queries/portfolioPeriods.graphql:14]: Even though this is here in the query, the Judges aren't actually attached in the db in any way whatsoever.

File: frontend/src/Admin/Page.js [hunk 3]
```diff
@@ -16,6 +17,7 @@ const Admin = () => (
     <Switch>
       <Route exact path='/' component={Dashboard} />
       <Route exact path='/show/new' component={CreateShow} />
+      <Route exact path='/portfolio/new' component={CreatePortfolioPeriod} />
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/Page.js:20]: @BrodrickS Here's the path to the new portfolio page.

File: frontend/src/Admin/queries/portfolioPeriods.graphql [hunk 4]
```diff
@@ -0,0 +1,17 @@
+query PortfolioPeriods {
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/queries/portfolioPeriods.graphql:1]: @BrodrickS This query grabs all the existing PortfolioPeriods. (Only works with the Admin by this PR)

File: frontend/src/Admin/queries/portfolioPeriods.graphql (hunk 2 above)
Comment (JohnPMurray) [reply frontend/src/Admin/queries/portfolioPeriods.graphql:14]: Do we have a ticket to do that later on?

**Review by JohnPMurray — APPROVED** [review]
Looks good. Keep in mind we also have to add edit functionality. That might be easier to do sooner rather than later since you kind of know whats going on, and since @gollb310 is working on the equivalent functionality for shows.

**Review by moiseslorap — APPROVED** [review]
good work

File: frontend/src/Admin/queries/portfolioPeriods.graphql (hunk 2 above)
Comment (johnbyrne727rit) [reply frontend/src/Admin/queries/portfolioPeriods.graphql:14]: We don't, Adding one now.

**Pull request merged**